public/app/core/components/Login/LoginServiceButtons.tsx

```diff
@@ -24,7 +24,7 @@ const loginServices: () => LoginServices = () => {
     saml: {
       bgColor: '#464646',
       enabled: config.samlEnabled,
-      name: 'SAML',
+      name: config.samlName || 'SAML',
```

Comment:
nit: maybe change to
```suggestion
      name: config.samlName ?? 'SAML',
```

Reply:
@hugohaggmark checking into your suggestion I found an issue with the backend that is preventing me from returning the default value to the frontend correctly, it's returning an empty string instead of the default value or null.

I was thinking it would be better to wait for grafana/grafana-enterprise#2021 to be fixed before I continue this PR so we don't need all this redundancy on the frontend, unless the redundancy is good, then we might need to leave it as `||` instead of `??` due to the backend issue.

Reply:
No worries, just a nit pick. Should have clarified that 👍